src/lammps.cpp

```diff
@@ -559,6 +564,12 @@ LAMMPS::LAMMPS(int narg, char **arg, MPI_Comm communicator)
     input->one(cmd);
     error->done();
   }
+
+  // used in Variable::random_seed()
+  if(me == 0){
+    // NOTE: better random generator for random_seed()?
+    srand(time(NULL));
```

Comment:
This will cause random numbers to be truly random, which is something you don't want as it destroys repeatability.

Reply:
This is my point. If I want  to repeat my simulation, I can use a predefined seed, but if I want to repeat an "experiment" with "random" particle positions, I can use randomseed().

Reply:
Could create a list of, say 500, prime numbers given an initial seed. If someone provides 0, you get the current behavior, and for any other number you get a list of semi-random numbers that is repeatable. Then every call to randomseed simply increments the location in the pre-compiled list.
You would also be able to add additional checks that make sure each prime number is unique.